ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -689,15 +690,21 @@ def process_dbt_descriptions(self, data_model_link: DataModelLink):
                 service_name, database_name, schema_name, table_name = fqn.split(
                     table_entity.fullyQualifiedName.__root__
                 )
-
                 data_model = data_model_link.datamodel
+                force_override = False
+                if (
+                    data_model.resourceType != DbtCommonEnum.SOURCE.value
+                    and self.source_config.dbtUpdateDescriptions
+                ):
+                    force_override = True
+
```

Comment:
this code would also override any description added manually by the user right? would that be expected?

Reply:
Yes that is expected. The goal of `dbtUpdateDescriptions` flag is to override already present descriptions and make the descriptions from dbt the source of truth.